build.sbt

```diff
@@ -20,8 +20,8 @@ val previousCirceOpticsVersion = "0.11.0"
 
 def priorTo2_13(scalaVersion: String): Boolean =
   CrossVersion.partialVersion(scalaVersion) match {
-    case Some((2, minor)) if minor < 13 => true
-    case _                              => false
+    case Some(2, minor) if minor < 13 => true
```

Comment:
Looks like this broke the tests because scalafmt removed the parentheses from the tuple.

```
[info] loading project definition from /home/runner/work/circe-optics/circe-optics/project
/home/runner/work/circe-optics/circe-optics/build.sbt:23: error: wrong number of arguments for pattern Some[(Long, Long)](value: (Long, Long))
    case Some(2, minor) if minor < 13 => true
             ^
sbt.compiler.EvalException: Type error in expression
[error] sbt.compiler.EvalException: Type error in expression
[error] Use 'last' for the full log.
[warn] Project loading failed: (r)etry, (q)uit, (l)ast, or (i)gnore? (default: r)
Error: Process completed with exit code 1.
```
https://github.com/circe/circe-optics/runs/6435623653?check_suite_focus=true#step:5:14